### DISCUSSION TITLE
Пара вопросов по настройкам

### DISCUSSION THREAD

**Discussion by Darkkys** [OGSR/OGSR-Engine#393] [Q&A] [answered]
1. как поменять качество текстур? В текущей версии данной настройки нет...

2. Можно ли отключить 3д прицелы?

**Comment by xrLil-Batya** [comment, marked as answer]
| Можно ли отключить 3д прицелы?
g_3d_scopes 1/0

**Reply by xrSimpAI** [reply]
Качество текстур менять в 2022 нет смысла

**Reply by Darkkys** [reply]
тем не менее автор hd models не советует ставить на максималку (из-за возможных крашей)

**Reply by xrSimpAI** [reply]
бред

**Comment by Darkkys** [comment]
g scope - в чем прописывается ? )
В конфигах не нашел..

**Reply by xrSimpAI** [reply]
то команда для консоли

**Reply by Darkkys** [reply]
а, а прописать это нигде нельзя?

**Reply by xrSimpAI** [reply]
ну в скриптах где-нибудь типа такого можно написать:
```lua
get_console():execute("g_3d_scopes on")
get_console():disable_command("g_3d_scopes")
```
в данном случае они будут включены всегда и через консоль это поменять нельзя будет.
